packages/@sanity/base/src/presence/PresenceTooltip.tsx

```diff
@@ -1,40 +1,37 @@
-// @todo: remove the following line when part imports has been removed from this file
-///<reference types="@sanity/types/parts" />
-
-import {Tooltip, TooltipPlacement} from 'part:@sanity/components/tooltip'
 import React from 'react'
+import {Tooltip, Placement, Card, Flex} from '@sanity/ui'
 import {UserAvatar} from '../components/UserAvatar'
 import {FormFieldPresence} from './types'
 
-import styles from './PresenceTooltip.css'
+import {TextWrapper} from './PresenceTooltip.styled'
 
 interface PresenceTooltipProps {
   children?: React.ReactElement
   items: FormFieldPresence[]
-  placement?: TooltipPlacement
+  placement?: Placement
 }
 
 export function PresenceTooltip(props: PresenceTooltipProps) {
   const {children, items, placement} = props
 
   const content = (
-    <div className={styles.root}>
+    <Card radius={2} padding={2}>
       {items.map((item) => (
-        <div className={styles.item} key={item.user.id}>
-          <div className={styles.avatarContainer}>
-            <UserAvatar user={item.user} status="online" />
-          </div>
+        <Card key={item.user.id} padding={1}>
+          <Flex align="center">
+            <div>
+              <UserAvatar user={item.user} status="online" />
+            </div>
 
-          <div className={styles.textContainer}>
-            <div className={styles.item__displayName}>{item.user.displayName}</div>
-          </div>
-        </div>
+            <TextWrapper>{item.user.displayName}</TextWrapper>
+          </Flex>
+        </Card>
       ))}
-    </div>
+    </Card>
   )
 
   return (
-    <Tooltip content={content} placement={placement}>
+    <Tooltip content={content} open placement={placement}>
```

Comment:
I believe `open` is not needed here?

Reply:
Good catch, this is a remnant from when I was trying the popovers out to debug a visual bug :) 
Will remove!